.travis.yml

```diff
@@ -24,12 +24,8 @@ matrix:
       env: PYTHON=python3 # I'm not sure how to install Python 3 on Travis OSX
 before_install:
     - test "x$TRAVIS_OS_NAME" = xosx -o "x$PYTHON" = xConda || (sudo apt-get -qq update && sudo apt-get install -y $PYTHON)
-script:
-    - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-    - julia -e 'Pkg.clone(pwd()); Pkg.build("PyCall")'
-    - julia -e 'import PyCall' # precompile
-    - julia -e 'Pkg.test("PyCall"; coverage=true)'
+
 after_success:
-    - julia -e 'cd(Pkg.dir("PyCall")); Pkg.add("Coverage"); using Coverage; Codecov.submit(process_folder())'
+    - julia -e 'using Pkg; cd(joinpath(dirname(pathof(PyCall)), "..")); Pkg.add("Coverage"); using Coverage; Codecov.submit(process_folder())'
```

Comment:
`using PyCall` is needed if you want the `pathof` it. But this `cd` can be removed completely.